controller/src/controller/models/domain/rsptag.py

```diff
@@ -305,13 +321,16 @@ def _from_match(
         if rest:
             display_name += f" [{rest}]"
 
+        image_date = cls._calculate_date(data)
+
         # Return the results.
         return cls(
             image_type=image_type,
             version=version,
             tag=tag,
             cycle=int(cycle) if cycle else None,
             display_name=display_name,
+            date=image_date,
```

Comment:
```suggestion
            date=cls._calculate_date(data)
```
and remove the `image_date` variable above.